### DISCUSSION TITLE
What do you want to use Documenso for?

### DISCUSSION THREAD

**Discussion by ElTimuro** [documenso/documenso#3] [General]
What do you want to use Documenso for? Tell us about where you want to use or integrate Documenso for signing. This way we can plan the best path forward 🚀

**Comment by yaacov-commenda** [comment]
Hey, my startup (commenda.io) is building some legal-tech features and we want a tighter integration than vendors like Docusign are willing to provide. We are considering building our own e-sign software, and looking into Documenso as an alternative. 


The following features are needed for parity with existing vendors like Docusign: 
- Admins/our team should be able to create new documents for signing, manage PDF upload and field placement, etc.
- We should be able to programatically create a new instance of an existing document (so call an API to send an email to several users and initiate a signature flow. 
- Signature flow will need multiple signers, initially this can probably be in parallel but we would ideally like support for a sequence of documents 

The following features would make you better than Docusign: 
- Signers should have accounts where they can manage all their previously-signed documents. Documenso should be configurable to call a third-party service for auth. 
- Full customizability of the frontend. Our initial use cases are: 
1. Redirecting to an external URL after sign-in 
2. Putting our own branding on the page in addition to any third-party branding 
3. Adding context to documents, like, an element that pops up on the side to explain some clause that might be confusing

**Comment by ElTimuro** [comment]
Hey, thanks for sharing. "tighter integration than vendors like Docusign" This is exactly what Documenso is about. Your features are, as far as I can tell, all on the internal roadmap. I'll take a look, when this could be usable in Prod. Feel free to share Details But this is pretty much how the Vision for the project looks like 🚀

**Comment by PeerRich** [comment]
we send a lot of NDAs, DPA and BAAs and we wanna automate it more.

ideally whitelabel and integrated into our product

**Reply by yangwao** [reply]
I would be customer for this as well with our company, having signing on monthly basis 30-60 docs is night mare.

**Comment by ivoflipse** [comment]
Working in the (software as a) medical device industry focusing on the US, we need to comply with FDA's [21 CFR Part 11](https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfcfr/CFRSearch.cfm?CFRPart=11).

I feel like there's always a level of distrust if we were to self-host such a system as one could commit fraud more easily when you administer the database yourself. But with a strong audit log and sound design, I think it should be doable to ["validate"](https://www.fda.gov/regulatory-information/search-fda-guidance-documents/general-principles-software-validation) such a tool.

With a hosted service, those concerns are likely not applicable, but there would be some concerns on how sure are we this service will stay alive long enough. If the plug gets pulled, what kind of records or data dump would suffice to show that historic signatures remain valid would be a concern.

Overall great work ❤️

**Reply by ElTimuro** [reply]
- We are currently looking into 21 CFR 11 compliance for a self-hosted case
- I would say this is easier achieved in a self-hosted way since the hoster can determine deploys and recertification after as well as complete data privacy
- As you said, using audit logs and solid permissions should be enough to enable self-host. The company itself can be regarded as a trusted entity for this case I would say, since the guarantee the CFR compliance. 
- We will probably offer this hosted at some point too, but much later than the self-hosted version I'd say, since it's a lot of compliance
- Progress and thought will be shared here among other places: https://github.com/documenso/documenso/issues/224

**Reply by ElTimuro** [reply]
Also thanks :)

**Comment by tpayne1984** [comment]
Great project idea.  We prototyped a Formstack + Docusign and ended up spinning a laravel application with DocRaptor for document generation and kind of put the signing on pause, but we tried a lot of signature services and I think this would be a great space for a paid service where the signatures were around $0.25 to $0.15 a signature.  (For us, our document costs are quite low and DOcusign is several dollars per signed document, and the costs just don't align).   Could I suggest a pay-per-document model -- $0.25 USD per document for hosted, we would be happy to pay for knowing the service is funded and will stick around.   Overall, we have found the API costs for signing to be a bit too high for our revenue per document.    We really like the DocRaptor pricing model though and if you guys wanted to get into this space,  I'd check out their approach.

**Reply by fmerian** [reply]
absolutely *love* this idea of a pay-per-document model! I +1

**Comment by padjakkels** [comment]
I would about 150 clients to sign a service agreement each year. I would like to host it on my own website if possible.

**Reply by ElTimuro** [reply]
- Nice
- Do you want to self host a complete version or embed a our hosted version?

**Reply by padjakkels** [reply]
Self hosting perhaps

**Comment by fmerian** [comment]
would love to use Documenso to sign the new contributor license agreement (CLA)

**Comment by astrakid** [comment]
hi, i would like to let customer in a doctor's office sign their data privacy approval without the need of printing the sheets and scan them afterwards. this seems to work with documenso, but i have to enter an email address. most of the patients don't even own an emailadress. would like to see documenso offering a workflow without need for email-adresses.
kind regards,
andre

**Reply by Novapixel1010** [reply]
This does work FYI.

**Reply by astrakid** [reply]
New Feature? IT didn't work when i tested IT the last time

**Reply by Novapixel1010** [reply]
You can upload your pdf then make a template and then you can make a direct link. You could take that direct link and just make it a QR code or setup some kind of tablet as a kiosk.

**OR**

 You could even provide a dummy email that all the sign docs go to after signed. 

> I might be adding this my tech guide.

**Comment by ivannovazzi** [comment]
For professional services firms (accountants, law firms, consultants), the use case often goes beyond pure e-signature: you need the client to **sign** an engagement letter *and* **upload** supporting documents (ID, financial statements, prior-year returns, etc.) in the same onboarding flow.

That gap — sign + collect supporting files together — is what we built [Gatherly](https://news.ycombinator.com/item?id=47085201) around. Clients get a single Secure Link to a branded portal where they sign with eIDAS-qualified signatures and drop their documents, without needing an account. Automated reminders chase them so the firm doesn't have to.

Documenso covers the signing side really well. For anyone evaluating whether they also need the document request layer on top, Gatherly might bridge that gap.
